docs/releases/temporary-versions/version-v12.0.0b1.md

```diff
@@ -0,0 +1,188 @@
+
+By default, InvenioRDM allows restricted records. However, with the addition of the RDM_ALLOW_RESTRICTED_RECORDS configuration variable, you can now choose to disallow restricted records if desired, and any attempt to create or update a record with restricted access will result in an error.
+
+To use this configuration variable, add it to your `invenio.cfg`:
+
+```
+RDM_ALLOW_RESTRICTED_RECORDS = False
+```
+
+With this configuration option set to True, users cannot create records with restricted access by setting the access field to "restricted". The user interface will no longer show the option to restrict the record, and the rest API will return an error if the user has no sufficient permissions to perform this action. This will disable the restricted access and the users will be able to create only open access records.
+
+
+### New export formats
+
+We are pleased to announce that the latest release includes several new export formats to facilitate sharing and dissemination of research data. These new export formats offer a wider range of options for exporting metadata, making it easier to integrate with various third-party systems and to meet different requirements for data publication. The new export formats are available in addition to the existing export formats, providing more flexibility and options for researchers and data managers. We believe that these new additions will help advance the goal of open science and data sharing, and we hope that they will be useful to the research community.
+
+Detailed documentation of export formats is available [here](../../reference/export_formats.md) 
+
+### Metadata extensions 
+
+The regular data model of InvenioRDM provides a set of core metadata fields that are commonly used to describe research data. These fields are designed to be generic enough to cover a wide range of research data, while still providing enough detail to describe the research in a meaningful way.
+
+However, there are situations where additional metadata fields are needed to capture more specific information about the research data. InvenioRDM provides the ability to add custom fields to the data model to accommodate these specific needs.
+
+The custom fields, such as journal, thesis, conference, imprint, and notes, are not included in the regular data model because they are not commonly used for describing research data. These fields are more specific to certain types of research or publications, and therefore, may not be relevant or necessary for all research data - however they are provided now provided possible extensions to the data model. Instance manages is able to include these fields by configuration. For more details on how to configure the aforementioned fields, visit [reference page](../../reference/custom_fields/fields_list.md).
```

Comment:
```suggestion
The custom fields, such as journal, thesis, conference, imprint, and notes, are not included in the regular data model because they are not commonly used for describing research data. These fields are more specific to certain types of research or publications, and therefore, may not be relevant or necessary for all research data - however they are now provided as possible extensions to the data model. Instance manages is able to include these fields by configuration. For more details on how to configure the aforementioned fields, visit [reference page](../../reference/custom_fields/fields_list.md).
```